src/app/core/config/dynamic-routing/submission-success/submission-success.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="success-container">
```

Comment:
Let's move this component to `features/public-form`. I think this is specific to that module only.